scoring/static/scoring/scss/table-question-council.scss

```diff
@@ -72,3 +72,33 @@ $question-types: (
         }
     }
 }
+
+// This class helps us avoid using Bootstrap display classes to hide/display elements, especially in tables where JS classes that handle display properties might conflict with the !important rules Bootstrap uses. With this class we wouldn't need to use d-none d-md-table-cell or d-md-block.
```

Comment:
FWIW, another way you could do this is by telling Bootstrap 5 to generate utility classes like `d-lg-revert`, which you could then use in the HTML.

You can do this by adding overriding the "display" section of the big `$utilities` map, the same way we add custom colours, font-weights, and widths in `scoring/static/scoring/scss/_utils.scss`. Eg: 

```scss
"display": map-merge(
    map-get($utilities, "display"),
    (
        values: inline inline-block block grid table table-row table-cell flex inline-flex none revert
    )
),
```

Just a thought!

Reply:
Yes, the problem with this approach and I why ended up creating that is because in some cases, when using the 2023 score switch the rules ended up conflicting when and element was also using classes like `js-previous-year-score`, because Bootstrap uses `!important` then some element that shouldn't be displayed were still being displayed. As an alternative I thought I could have added the bootstrap classes and add specific rules `js-previous-year-score-display d-lg-table-row` situations.